### DISCUSSION TITLE
single point energe calculate

### DISCUSSION THREAD

**Discussion by Edinbur** [deepmodeling/dpgen#1532] [Q&A]
Is it possible to set the wave function to be read when calling vasp for single point energy calculation in dpgen? That is, set the path of wave function in param.json, otherwise the single point energy calculation will be slow if the system is big.

**Comment by njzjz** [comment]
Contribution is welcome.

**Comment by Edinbur** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
